packages/react/src/components/ColorPicker/ColorPicker.test.tsx

```diff
@@ -110,89 +99,99 @@ describe('ColorPicker', () => {
+    verifyInputs(inputs, AEAEAE);
     // shouldn't call onChange when the consumer updates the color prop
     expect(onChange).toHaveBeenCalledTimes(0);
   });
 
   it('ignores invalid updates to color prop', () => {
-    wrapper = mount(<ColorPicker color={abcdef} onChange={onChange} componentRef={colorPickerRef} />);
+    const { getAllByRole, rerender } = render(
+      <ColorPicker color={abcdef} onChange={onChange} componentRef={colorPickerRef} />,
+    );
 
-    wrapper.setProps({ color: 'foo' });
+    rerender(<ColorPicker color={'foo'} onChange={onChange} componentRef={colorPickerRef} />);
     expect(colorPicker!.color.hex).toEqual(abcdef.hex);
-    verifyInputs(abcdef);
+    const inputs = getAllByRole('textbox') as HTMLInputElement[];
+    verifyInputs(inputs, abcdef);
     expect(onChange).toHaveBeenCalledTimes(0);
   });
 
   it('hides alpha control slider', () => {
-    wrapper = mount(<ColorPicker color={white} alphaType="none" />);
+    const { getAllByRole, queryByRole } = render(<ColorPicker color={white} alphaType="none" />);
 
-    const alphaSlider = wrapper.find('.is-alpha');
-    const tableHeaders = wrapper.find('thead td');
+    const alphaSlider = queryByRole('slider', { name: 'Alpha' });
+    const tableHeaders = getAllByRole('group')[1].firstElementChild!.firstElementChild!.children;
 
     // There should only be table headers and inputs for hex, red, green, and blue (no alpha)
-    expect(alphaSlider.exists()).toBe(false);
+    expect(alphaSlider).toBeFalsy();
     expect(tableHeaders).toHaveLength(4);
-    verifyInputs(white, 'none');
+    const inputs = getAllByRole('textbox') as HTMLInputElement[];
+    verifyInputs(inputs, white, 'none');
   });
 
   it('shows preview box', () => {
-    wrapper = mount(<ColorPicker color="#FFFFFF" showPreview={true} />);
+    const { getAllByRole } = render(<ColorPicker color="#FFFFFF" showPreview={true} />);
 
-    const previewBox = wrapper.find('.is-preview');
+    const previewBox = getAllByRole('group')[0].querySelector('.is-preview');
 
     // There should be one preview box
-    expect(previewBox.exists()).toBe(true);
+    expect(previewBox).toBeTruthy();
   });
 
   it('hides preview box', () => {
-    wrapper = mount(<ColorPicker color="#FFFFFF" showPreview={false} />);
+    const { getAllByRole } = render(<ColorPicker color="#FFFFFF" showPreview={false} />);
 
-    const previewBox = wrapper.find('.is-preview');
+    const previewBox = getAllByRole('group')[0].querySelector('.is-preview');
 
     // There should be one preview box
-    expect(previewBox.exists()).toBe(false);
+    expect(previewBox).toBeFalsy();
   });
 
   it('renders default RGBA/Hex strings', () => {
-    wrapper = mount(<ColorPicker color="#FFFFFF" />);
+    const { getAllByRole } = render(<ColorPicker color="#FFFFFF" />);
 
-    const tableHeaders = wrapper.find('thead td');
+    const tableHeaders = Array.from(getAllByRole('group')[1].firstElementChild!.firstElementChild!.children);
```

Comment:
I think actually `container.querySelectorAll('thead td')` would be preferable in this case. Same for other similar queries.